src/domain/settlement/index.js

```diff
@@ -83,8 +84,7 @@ module.exports = {
         participants
       }
     } else {
-      const err = new Error('2001 TODO Settlement not found')
-      throw err
+      throw ErrorHandler.Factory.createInternalServerFSPIOPError('2001 TODO Settlement not found')
```

Comment:
Is the API spec going to be updated to include settlements? If not maybe 3200, Generic ID not found could be used?

Reply:
I think we have to use Scheme Specific errors here @ndonnan , @gibaros .. We'll not be able to add these to the FSPIOP Spec as that doesn't include Settlements aspects. However, we can have new errors we come up with, in the Settlements API itself. For example, 6xxx errors can be for Settlements related issues (since 1xxx - 5xxx are used in the FSPIOP API).

How about 6001 - _A Settlement with the given Settlement ID is not found_.

But this should be well documented not just in the docs but in the API itself, I think.. I will create an issue for this and follow-up.. Lets not keep the current PR blocked because of that issue.

Reply:
Created an issue here: https://github.com/mojaloop/project/issues/928

Reply:
Ok cool, makes sense